### PR TITLE
Pull Request for Initial publication of Homicide Offences dataset

### PR DESCRIPTION
**Pull request by santhosh-thangavel**
Added new qube-config file
Added warngling.py
Added a new csv file
Added metadata as well
Added Marker column for missing Observation values

#28

### CODE REVIEW

**Review by abdulkasim1 — CHANGES_REQUESTED** [review]

File: homicide-offences/homicide-offences.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Homicide Offences (Rate per million population)",
+    "creator": "https://www.gov.uk/government/organisations/office-for-national-statistics",
+    "summary": "Homicide",
+    "description": "## Rate of Homicide offences per million population.  \nThe indicator is: Homicide.\nThe Levelling Up category is: Restoring a sense of community, local pride and belonging.  \nThe Mission is 11(expand with information on what mission 11 means).\nThe nation covered is England and Wales.",
```
[the remark this answers is not in the document]
Comment (santhosh-thangavel) [reply homicide-offences/homicide-offences.json:6]: Added "Rate of Homicide offences per million population in England and Wales" for summary

File: homicide-offences/homicide-offences.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        "subnational",
+        "rate",
+        "offences",
+        "Homicide",
+        "united-kingdom"
+    ],
+    "publisher": "https://www.gov.uk/government/organisations/office-for-national-statistics",
+    "themes": [
+        "sub-national-data"
+    ],
```
[the remark this answers is not in the document]
Comment (santhosh-thangavel) [reply homicide-offences/homicide-offences.json:18]: Added "https://www.ons.gov.uk/peoplepopulationandcommunity/crimeandjustice" URL for themes

File: homicide-offences/homicide-offences.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        },
+        "Area": {
+            "from_template": "statistical-geography"
+        },
+        "Mission": false, 
+        "Category": false,   
+        "Indicator": false,
+        "Observation":{
+            "type": "observations",
+            "measure":{
```
[the remark this answers is not in the document]
Comment (santhosh-thangavel) [reply homicide-offences/homicide-offences.json:43]: "data_type" : "float" is added now

File: homicide-offences/homicide-offences.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    "themes": [
+        "sub-national-data"
+    ],
+    "columns": {
+        "Period": {
+            "type": "dimension",
+            "code_list": {
+                "title": "Gregorian Year",
+                "summary": "Gregorian Year for homicide offences in England and Wales",
+                "from_existing": "http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
```
[the remark this answers is not in the document]
Comment (santhosh-thangavel) [reply homicide-offences/homicide-offences.json:25]:
```
"title":"Period"
"summary":"Period for Homicide offences in England and Wales"
```
is added

File: homicide-offences/homicide-offences.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+        "Mission": false, 
+        "Category": false,   
+        "Indicator": false,
+        "Observation":{
+            "type": "observations",
+            "data_type": "float",
+            "measure":{
+                "label":"Crime Rate (rate per million population)",
+                "from_existing": "https://purl.org/csv-cubed/resources/measures#crime-rate"
+            },
```
Comment (abdulkasim1) [inline homicide-offences/homicide-offences.json:47]: Change label to just "Crime Rate".

**Review by abdulkasim1 — CHANGES_REQUESTED** [review]

File: homicide-offences/homicide-offences.json (hunk 5 above)
Comment (santhosh-thangavel) [reply homicide-offences/homicide-offences.json:47]: Added label `"Crime Rate"`

**Review by abdulkasim1 — APPROVED** [review]

**Pull request merged**